lib/adyen/client.rb

```diff
@@ -125,7 +125,7 @@ def call_adyen_api(service, action, request_data, headers, version)
       end
 
       # add application only on checkout service
-      if service == 'Checkout' || service == 'CheckoutUtility'
+      if action == "payments"
```

Comment:
it should be used also for PAL payment, Checkout paymentLinks and modifications, is it creating problems?

Reply:
I think the latest version of /payments/details is doing some unusual validation on fields(got another issue on python), let's discuss separately.
Your approach looks fine, can you also add `service == 'Payment'` to the OR? since the old PAL /payment also works with appinfo

Reply:
After checking the API explorer I found that paymentLinks also support it but we should be specific where we send the applicationInfo because the Checkout endpoints are going to validate all the request fields and throw an error if an unsupported field is sent.